app/models/changeset.rb

```diff
@@ -200,6 +200,8 @@ def save_with_tags!
     end
   end
 
+  def comment = tags["comment"].to_s.presence
```

Comment:
Is the `.to_s` required here? Just wondering, I'm not sure. If the value is nil, `#presence` will have the same behaviour, unless I'm much mistaken.

I appreciate this is just copied from the existing code, but since we are improving this, we might as well consider this question.